master/reference/node/configuration.md

```diff
@@ -0,0 +1,32 @@
+| IP | The IPv4 address to assign this host. When specified, the address is saved in the [node resource configuration]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/node) for this host, overriding any previously configured value. When omitted, if an address **has** been saved in the node resource, then that value will be used. When omitted, if an address **has not** yet been configured in the node resource, the node will auto-detect an IPv4 address and configure the node resource with that address. This autodetection can be forced (even if a value has already been set in the node resource) by setting IP to "autodetect". Doing so will overwrite any value configured in the node resource. | ip address | |
+| IP6 | The IPv6 address for Calico will bind to. When specified, the address is saved in the  [node resource configuration]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/node) for this host, overriding any previously configured value. When omitted, if an address has not yet been configured in the node resource, IPv6 routing is not enabled. When omitted, if an IPv6 address has been previously configured in the node resource, IPv6 is enabled using the already configured address. | ipv6 address | |
+| AS | The AS number for this node. When specified, the value is saved in the node resource configuration for this host, overriding any previously configured value. When omitted, if an AS number has been previously configured in the node resource, that AS number is used for the peering.  When omitted, if an AS number has not yet been configured in the node resource, the node will use the global value (managed through `calicoctl config set/get asnumber`). | int | global AS number |
+| DATASTORE_TYPE | Type of datastore. | kubernetes, etcd | etcd |
+| WAIT_FOR_DATASTORE | Wait for connection to datastore before starting. If a succesfull connection is not made, node will shutdown. | boolean | false |
+| CALICO_LIBNETWORK_ENABLED | Enables running the docker-libnetwork plugin directly in the calico/node container. | boolean | true |
+| CALICO_LIBNETWORK_IFPREFIX | Interface prefix to use for the network interface within the Docker containers that have been networked by the Calico driver. | string | cali |
+| CALICO_NETWORKING_BACKEND | Describes which BGP networking backend to use| gobgp, bird, none | bird |
+| **etcd configuration** | | | |
+| ETCD_ENDPOINTS    | A comma separated list of etcd endpoints (optional, defaults to http://127.0.0.1:2379) | http://etcd1:2379 | |
+| ETCD_KEY_FILE     | Path to the etcd key file (optional)       | /etc/calico/key.pem | |
+| ETCD_CERT_FILE    | Path to the etcd client cert (optional)    | /etc/calico/cert.pem | |
+| ETCD_CA_CERT_FILE | Path to the etcd CA file (optional)        | /etc/calico/ca.pem | |
+
+The ETCD_CA_FILE, ETCD_CERT_FILE, and ETCD_KEY_FILE
+environment variables are required when using etcd with SSL/TLS.  The values
```

Comment:
"The values provided here... template"  what template?  I think this whole paragraph should go.